Common/ModdedPlayer.cs

```diff
@@ -16,326 +16,263 @@
+				items.Add(new Item(ModContent.ItemType<SynergyEnergy>()));
+			}
+			if (ModContent.GetInstance<BossRushModConfig>().Nightmare)//gitgudlol
+			{
+				items.Add(new Item(ModContent.ItemType<PremiumCardPacket>()));
+				items.Add(new Item(ItemID.RedPotion, 10));
+			}
+			if (Player.name == "LQTXinim" || Player.name == "LowQualityTrashXinim") {
+				items.Add(new Item(ModContent.ItemType<RainbowTreasureChest>()));
+			}
+			if (Player.name == "FeelingLucky") {
+				items.Add(new Item(ModContent.ItemType<GodDice>()));
+			}
+			if (Player.name.ToLower().Trim() == "drugaddict") {
+				items.Add(new Item(ModContent.ItemType<WonderDrug>(), 99));
+			}
+			if (Player.name.Contains("Ninja")) {
+				items.Add(new Item(ItemID.Katana));
+				items.Add(new Item(ItemID.Shuriken, 100));
+				items.Add(new Item(ItemID.ThrowingKnife, 100));
+				items.Add(new Item(ItemID.PoisonedKnife, 100));
+				items.Add(new Item(ItemID.BoneDagger, 100));
+				items.Add(new Item(ItemID.FrostDaggerfish, 100));
+				items.Add(new Item(ItemID.NinjaHood));
+				items.Add(new Item(ItemID.NinjaShirt));
+				items.Add(new Item(ItemID.NinjaPants));
+				items.Add(new Item(ModContent.ItemType<GuideToMasterNinja>()));
+				items.Add(new Item(ModContent.ItemType<GuideToMasterNinja2>()));
+			}
+			if (Player.name == "HMdebug") {
+				items.Add(new Item(ModContent.ItemType<IronLootBox>()));
+				items.Add(new Item(ModContent.ItemType<SilverLootBox>()));
+				items.Add(new Item(ModContent.ItemType<GoldLootBox>()));
+				items.Add(new Item(ModContent.ItemType<CorruptionLootBox>()));
+				items.Add(new Item(ModContent.ItemType<CrimsonLootBox>()));
+				items.Add(new Item(ModContent.ItemType<IceLootBox>()));
+				items.Add(new Item(ModContent.ItemType<HoneyTreasureChest>()));
+				items.Add(new Item(ModContent.ItemType<LootboxLordSummon>()));
+				items.Add(new Item(ModContent.ItemType<PerkChooser>(), 8));
+				items.Add(new Item(ItemID.LifeCrystal, 15));
+				items.Add(new Item(ItemID.ManaCrystal, 4));
+				items.Add(new Item(ItemID.KingSlimeBossBag));
+				items.Add(new Item(ItemID.EyeOfCthulhuBossBag));
+				items.Add(new Item(ItemID.EaterOfWorldsBossBag));
+				items.Add(new Item(ItemID.BrainOfCthulhuBossBag));
+				items.Add(new Item(ItemID.SkeletronBossBag));
+				items.Add(new Item(ItemID.QueenBeeBossBag));
+				items.Add(new Item(ItemID.DeerclopsBossBag));
+				items.Add(new Item(ItemID.GuideVoodooDoll));
+			}
+			if (Player.name == "I want to die" || Player.name == "Give me hell") {
+				items.Clear();
+				items.Add(new Item(ModContent.ItemType<WoodenLootBox>()));
+				items.Add(new Item(ModContent.ItemType<CursedSkull>()));
+				items.Add(new Item(ModContent.ItemType<CardPacket>()));
+				items.Add(new Item(ModContent.ItemType<PowerEnergy>()));
+			}
+			if (Player.IsDebugPlayer()) {
+				items.Add(new Item(ModContent.ItemType<ModStatsDebugger>()));
+				items.Add(new Item(ModContent.ItemType<ShowPlayerStats>()));
+			}
+			return items;
```

Comment:
It looks like you could just use `yield return` instead, so compiler takes care of this instead.